app/javascript/bundles/User/components/DonationsHistory.jsx

```diff
@@ -27,17 +27,28 @@ const DONATION_TYPES = {
   SUBSCRIPTION: 'Monthly Subscription'
 }
 
+const calcDonationTotal = donations => {
+  return donations.reduce((acc, donation) => (acc += donation), 0)
```

Comment:
@castrolem this is fine for now since we won't have many donations at the beginning. A better approach will be calculating this in the backend

Reply:
All the data is there and this is a performant operation on browsers, shouldn't be bad even with lots of donations.